doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -54,6 +57,272 @@ Additionally, maintaining ABI compatibility allows the nRF54H20 SoC binary compo
+  This replaces the temporary IPC mechanism from the last release.
+
+Fixed
+-----
+
+* An issue that set the CTRLAP.BOOTSTATUS firmware sequence number always to zero. (NCSDK-33265)
+
+IronSide Secure Element (IronSide SE) v20.1.0
+=============================================
+
+Added
+-----
+
+* Added experimental support for a limited :ref:`PSA Crypto API <ug_psa_certified_api_overview_crypto>` service.
+  This is built on top of a temporary IPC mechanism which will soon be replaced.
+  The top-level interface will remain the same. (NCSDK-32163)
+
+  The PSA Crypto API support through the :ref:`ug_crypto_architecture_implementation_standards_ironside` is currently limited to the following ``PSA_WANT`` symbols for :ref:`cryptographic feature selection <crypto_drivers_feature_selection>`:
+
+  * ``PSA_WANT_GENERATE_RANDOM``
+  * ``PSA_WANT_ALG_SHA_256``
+  * ``PSA_WANT_ALG_SHA_512``
+  * ``PSA_WANT_ALG_GCM``
+  * ``PSA_WANT_ALG_ECDSA``
+  * ``PSA_WANT_ALG_ECDH``
+  * ``PSA_WANT_ALG_ED25519PH``
+  * ``PSA_WANT_ECC_SECP_R1_256``
+  * ``PSA_WANT_KEY_TYPE_ECC_PUBLIC_KEY``
+  * ``PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_IMPORT``
+  * ``PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_EXPORT``
+  * ``PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_GENERATE``
+  * ``PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_DERIVE``
+
+  To use the service, set the :kconfig:option:`CONFIG_NRF_SECURITY` to ``y``.
+  For more information on the implementations available in the cryptographic drivers, see the :ref:`crypto_drivers`.
+
+  .. note::
+     The v20.1.0 support for this PSA Certified Crypto API is not compatible with |NCS| v3.0.0 or v3.1.0.
+     It is only compatible with the ``sdk-nrf`` revision ``1b2abb07b8b2528ecaf86f54e0c6cf48c425055a``.
+
+Updated
+-------
+
+* ``EXTRAVERSION`` is now included in ``SICR.TBS.x.VERSION``.
+
+IronSide Secure Element (IronSide SE) v20.0.1
+=============================================
+
+Updated
+-------
+
+* AUX-AP to be always disabled.
+* Internal optimization of MRAM and RAM usage.
+
+Removed
+-------
+
+* Initializing TDD on system boot as it is not retained.
+* Initializing P6 and P7 pins to be EXMIF/TPIU to prevent unacceptable output states.
+
+IronSide Secure Element (IronSide SE) v20.0.0
+=============================================
+
+This is the first release that is based on the new Secure Domain firmware architecture.
+Most of the functionality from the preceding version of SDFW has been disabled or removed and will be gradually reintroduced in upcoming versions.
+
+Added
+-----
+
+* Support for the ``ERASEALL`` command through the boot command interface. (NCSDK-31997)
+* Support for the ``DEBUGWAIT`` command through the boot command interface.
+* A new scheme for status reporting through the ``BOOTSTATUS`` register in CTRL-AP. (NCSDK-32355)
+
+Updated
+-------
+
+* SCFW to be included in the URoT firmware partition.
+  Additionally, the SysCtrl CPU is always started. (NCSDK-31993)
+* SDFW to not start the Radio core.
```

Comment:
```suggestion
* SDFW to not start the radio core.
```

Reply:
As above, capitalized in the SoC docs: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_architecture_cpu.html